### ISSUE TITLE
Bug with glass and water not showing through glass

### ISSUE THREAD

**Issue by LeFogel** [sp614x/optifine#5390]
![2021-02-27_18 33 40](https://user-images.githubusercontent.com/79767845/109399492-69385000-7943-11eb-8de0-461db1213747.png)
![2021-02-27_18 33 17](https://user-images.githubusercontent.com/79767845/109399495-6c334080-7943-11eb-8a27-45d778dd535c.png)

I checked without optifine and it disappeared so its an issue with optifine.
how to reproduce: place a bunch of layers of water and glass (in my case it was white stained glass with connected textures and I turned render regions off)

**Comment by Regnander** [comment]
Related #5107

**Comment by LeFogel** [comment]
Just to be clear, I have also experienced the #5107 bug and this one is different because it happens no matter oif render regions is on or off.

**Comment by LeFogel** [comment]
I am also using the newest version of Optifine.

**Comment by Regnander** [comment]
> I am also using the newest version of Optifine.

And that would be G7, like in the screenshot?

The reason I'm asking is because I don't know the last time you checked the website for any new version.

**Comment by KaiAF** [comment]
Do you have any resource pack enabled? If so, what is it?

**Comment by LeFogel** [comment]
> Do you have any resource pack enabled? If so, what is it?

Yes, but none of it affects glass or water, its the vannila tweaks pack with some changes to the ui and 3d models for blocks.

**Comment by LeFogel** [comment]
> > I am also using the newest version of Optifine.
> 
> And that would be G7, like in the screenshot?
> 
> The reason I'm asking is because I don't know the last time you checked the website for any new version.

Yes it is G7

**Comment by LeFogel** [comment]
> Do you have any resource pack enabled? If so, what is it?

Is more information needed? You haven't removed the tag.

**Comment by KaiAF** [comment]
Sorry, hadn't had the time to go back and continue bug testing this. 
Does this only happen when a resource pack is enabled? I know you said it doesn't affect glass or water

**Comment by LeFogel** [comment]
> Sorry, hadn't had the time to go back and continue bug testing this.
> Does this only happen when a resource pack is enabled? I know you said it doesn't affect glass or water

It also happen without the resourcepack.

**Comment by KaiAF** [comment]
Okay thank you, are you able to tell me your current video settings? Maybe that can help me figure out how to reproduce it

**Comment by LeFogel** [comment]
> Okay thank you, are you able to tell me your current video settings? Maybe that can help me figure out how to reproduce it

I tested all of the settings, none of them change anything. I did figure out what causes it: you CAN see water through glass, just not stained glass, turning connected textures off makes it more consistent.
